pkg/services/validationService.go

```diff
@@ -476,3 +474,17 @@ func getClusterRequestMetadata(cliEvaluationId int, token string, skipped bool,
 
 	return clusterRequestMetadata
 }
+
+func getWarningCTABasedOnPassedPolicyCheck(isFailedPolicyCheck bool, cliEvaluationId int, registrationURL string) string {
+	baseUrl := "https://app.datree.io"
+
+	if strings.Contains(registrationURL, "staging") {
```

Comment:
I don't like this logic, seems too complicated/fragile

maybe instead of an `if` statement, trim everything after the "datree.io"?